src/editor/editor.cpp

```diff
@@ -166,7 +168,7 @@ namespace Editor
     TilesetLoad()
     {
         int req_format = STBI_rgb_alpha;
-        image_file_path = assets_rel_path_prefix + image_file_path;
+        image_file_path = assets_rel_path_prefix.string() + image_file_path;
```

Comment:
I think we should probably have 3 separate variables for:
- the text entered in the imgui text field
  - What the user sees 
- The assetprefix
- The full path name (assetprefix + imagepath)
  - will be used to load the image 


Reason is that currently this will add the "assets/" prefix in the imgui text field

One additional thing might be to add the prefix as a text item before the text field to make it more obvious it is from the assets folder
![Screenshot_20221004_124252](https://user-images.githubusercontent.com/47202511/193877885-11d68112-acc2-4829-9c5f-62cac4b6b8e4.png)